tools/flight_cli/client.py

```diff
@@ -0,0 +1,80 @@
+import json
+import logging
+import socket
+import sys
+
+import requests
+
+BUFFER_SIZE = 1024
+
+
+def gen_req(s, command, **kwargs):
+    message = json.dumps({"command": command, "meta": kwargs})
+    s.send(message.encode())
+
+
+def main():
+    if len(sys.argv) < 3:
+        logging.error("Please specify a host and port")
+        sys.exit(1)
+
+    TCP_HOST = sys.argv[1]
+    TCP_PORT = int(sys.argv[2])
+
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.connect((TCP_HOST, TCP_PORT))
+
+    while True:
+        params = input("> ").split()
+        try:
+            command = params[0]
+            logging.info(command)
+            if command == "exit":
+                gen_req(s, command)
+                s.shutdown(socket.SHUT_WR)
+                s.close()
+                return
+            elif command == "land":
```

Comment:
Can you put a comment or docstring somewhere that lists out the available commands and the order of parameters expected for each command?